File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.c
int findKthPositive(int* arr, int arrSize, int k){
    int i=1,pos=0;
        int n = arrSize;
        while(pos<n && k>0){
            if(arr[pos]==i){
                i++;
                pos++;
            }else{
                k--;
                i++;
            }
        }
        i--;
        return i+k;

}

File: 1539-kth-missing-positive-number/1539-kth-missing-positive-number.py
class Solution:
    def findKthPositive(self, arr: List[int], k: int) -> int:
        pos=0
        i=1
        n=len(arr)
        while k>0 and pos<n:
            if i==arr[pos]:
                i+=1
                pos+=1
            else:
                k-=1
                i+=1
        i-=1
        return i+k
        
        